ci/taskcluster/bin/decision.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+import os
+
+import requests
+from taskcluster import fromNow
+from taskcluster.utils import stableSlugId, dumpJson
+
```

Comment:
I'm not sure why, but flake8 doesn't care about this space. I even ran flake8 manually with `--isolated` (which makes it ignore config files). The only problems it found are line length problems.

-